app/views/admin/roles/new.html.erb

```diff
@@ -1,11 +1,16 @@
-<% page_title "New role" %>
-<div class="row">
-  <section class="col-md-8">
-    <h1>New role</h1>
-    <p class="warning">Do not create ministerial roles without consulting GDS.</p>
-    <%= render partial: "legacy_form", locals: {role: @role, show_consult_gds_warning: true} %>
-  </section>
-  <div class="col-md-4">
-    <%= legacy_simple_formatting_sidebar %>
+<% content_for :page_title, "New #{@role.name}" %>
```

Comment:
i don't think role will have a name here so it would just say "New". 

We can just have "New role"

Reply:
Yes of course, good spot. Thank you.